internal/metrics/smbstatus_test.go

```diff
@@ -602,3 +608,28 @@ func TestParseSMBStatusLocks(t *testing.T) {
 	assert.Equal(t, lock2.FileID.Inode, int64(52))
 	assert.Equal(t, lock2.NumPendingDeletes, 2)
 }
+
+func TestParseSMBStatusOpenFiles(t *testing.T) {
+	status, err := parseSMBStatus(smbstatusOutput5)
+	assert.NoError(t, err)
+	assert.Equal(t, len(status.OpenFiles), 2)
+	for _, openFile := range status.OpenFiles {
+		for _, open := range openFile.Opens {
+			oplock := open.OpLock
+			lease := open.Lease
+			if oplock.Lease {
+				assert.Equal(t, oplock.Exclusive, true)
+				assert.Equal(t, lease.Handle, false)
+				assert.Equal(t, lease.Read, true)
+				assert.Equal(t, lease.Write, true)
+				assert.Equal(t, lease.Text, "RW")
+			} else {
+				assert.Equal(t, oplock.Exclusive, false)
+				assert.Equal(t, lease.Handle, false)
+				assert.Equal(t, lease.Read, false)
+				assert.Equal(t, lease.Write, false)
+				assert.Equal(t, lease.Text, "")
```

Comment:
To cover other `SMBStatusOpenOpLock` types:
```
assert.Equal(t, oplock.Batch, true)
assert.Equal(t, oplock.LevelII, false)
assert.Equal(t, oplock.Text, "BATCH")
```

Reply:
Done

Reply:
If you meant to include the additional checks I don't see it in the updated version.